doc/tutorials/outsourced-cluster.rst

```diff
@@ -0,0 +1,106 @@
+##################################
+Outsourced Distributed Computation
+##################################
+
+Secure XGBoost is tailored toward an outsourced computation model -- one in which a client with sensitive data wants to outsource computation on the sensitive data to an untrusted server with a trusted hardware enclave. With a lot of training data, a cluster of machines, as opposed to a single server, may have to be used. This tutorial provides an example of such a scenario. 
+
+In this example, the client will encrypt data and send it to each machine in the cluster. The data remains encrypted until it is loaded inside each machine's enclave, at which point it can be decrypted. The cluster will then collectively train a model on the training data following the distributed training pattern outlined in the :doc:`Distributed Secure XGBoost tutorial <distributed>`. Lastly, an encrypted version of the trained model will be saved, available for the client to fetch. 
```

Comment:
This seems to suggest that the data is replicated across each machine, instead of being distributed... Rephrase / clarify?

Reply:
I thought the data is replicated across all machines? The distributed part just tells each node which partition of the data to train on, but the actual data itself exists as a whole on every machine, i.e. the data does not exist only on one machine.

Reply:
Hmm, I see what you mean. Either that or it's stored in a single location accessible to all the enclaves. How about something along the lines of: the client sends the encrypted data to the cluster, and each enclave loads and processes a part of the encrypted data?

Reply:
Done!